src/effects.cpp

```diff
@@ -8,6 +8,39 @@ Effects::Effects() {}
 
 Effects::~Effects() {}
 
+bool Effects::effectExists(effectNS::EFFECT_TYPE effectType) {
+	for (effectDurationPair curr : currentEffects) {
+		if (curr.first == effectType)
+			return true;
+	}
```

Comment:
Cannot access directly by key because using `enums` as key, `.find()` does not work here